tests/x11/shutdown.pm

```diff
@@ -20,6 +20,13 @@ use utils;
 
 sub run() {
     my $self = shift;
+    x11_start_program('xterm');
+    script_sudo(q{echo 'ForwardToConsole=yes' >> /etc/systemd/journald.conf});
+    script_sudo(q{echo 'MaxLevelConsole=debug' >> /etc/systemd/journald.conf});
+    script_sudo(qq{echo 'TTYPath=/dev/$serialdev' >> /etc/systemd/journald.conf});
+    script_sudo(q{systemctl restart systemd-journald});
```

Comment:
If you look at a random serial0.txt you will see that there is no journal, we only have kernel console log there